wayland-client/src/lib.rs

```diff
@@ -63,6 +64,15 @@ pub trait Proxy: Sized {
     /// Access the user-data associated with this object
     fn data<U: Send + Sync + 'static>(&self) -> Option<&U>;
 
+    /// Access the raw data associated with this object.
+    ///
+    /// For objects created using the scanner-generated interfaces, this will be an instance of the
```

Comment:
```suggestion
    /// For objects created using the scanner-generated methods, this will be an instance of the
```

Let's write it like this, "interface" is already a pretty overloaded term in wayland context.